packages/app-runtime/src/AppRuntime.ts

```diff
@@ -256,7 +256,9 @@ export class AppRuntime extends Runtime<AppConfig> {
         const moduleConstructor = AppRuntime.moduleRegistry[moduleConfiguration.location];
         if (!moduleConstructor) {
             const error = new Error(
-                `Module '${this.getModuleName(moduleConfiguration)}' could not be loaded, because it was not registered. Please register all modules before running init.`
+                `Module '${moduleConfiguration.location}' could not be loaded, because it was not registered. Please register all modules before running init. Available modules: ${Object.keys(
```

Comment:
Should probably use the (display) name.

Reply:
nope, we don't have the display name at this point. This is the only ugly thing with my approach ;)